app/views/links/index.slim

```diff
@@ -21,6 +21,9 @@
               a href="#{link.url}" target="_blank" rel="nofollow" #{link.title}
               br
               a href="#{link.url}" target="_blank" rel="nofollow" #{link.url}
+              - if link.time_to_read
+                br
```

Comment:
sorry but can we 
1) leave this on the same line with link text
2) keep spaces between words
3) remove link from this estimated time value ?